node/node.go

```diff
@@ -1404,7 +1404,6 @@ func (n *Node) OnStart() error {
 
 // OnStop stops the Node. It implements service.Service.
 func (n *Node) OnStop() {
-	n.BaseService.OnStop()
```

Comment:
What was your reasoning for removing this?

Reply:
The BaseService.OnStop() looks like doing nothing. 
https://github.com/tendermint/tendermint/blob/05bc2f344a20bd25beb4b6f9e6f41d2295d99492/libs/service/service.go#L180

And the `func (n *Node) OnStop()` be called by https://github.com/tendermint/tendermint/blob/05bc2f344a20bd25beb4b6f9e6f41d2295d99492/libs/service/service.go#L159

Reply:
Yeah it seems like you're right. I'm unsure why it was there in the first place then. I notice that `BaseService.OnStop()` get's called in other places as well.

Reply:
It looks weird when I was looking at the OnStop call. So I thought maybe this line has been added accidentally.